pkg/executables/kubectl.go

```diff
@@ -53,6 +53,16 @@ type Kubectl struct {
 	Executable
 }
 
+func (k *Kubectl) SearchCloudStackMachineConfig(ctx context.Context, name string, kubeconfigFile string, namespace string) ([]*v1alpha1.CloudStackMachineConfig, error) {
```

Comment:
Does it make sense for `Kubectl` to have these methods? I.e. will they ever be implemented?